index.html

```diff
@@ -60,5 +61,387 @@ <h2>I am a Software Developer</h2>
             </ul>
           </div>
         </section>
+        <section class="works">
+          <div class="grid">
+            <div class="gridItem grid1 d-flex flex-center">
+              <h2>My Recent Works</h2>
+              <img alt="line" src="img/line1.png" />
+            </div>
+            <div class="gridItem grid2 d-flex flex-center">
+              <img alt="project images" src="img/ImgPlaceholder.png" />
+              <h2>Multi-Post Stories</h2>
+              <p>
+                A daily selection of privately personalized<br />
+                reads; no accounts or sign-ups required.<br />
+                has been the industry's standard dummy<br />
+                text ever since the 1500s, when an<br />
+                unknown printer took a standard dummy <br />text.
+              </p>
+              <ul class="d-flex">
+                <li>
+                  <a href="#">
+                    <img alt="cssButton" src="img/cssTag.png" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img alt="htmlButton" src="img/htmlTag.png" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img alt="bootstrapButton" src="img/bootstrapTag.png" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img alt="rubyButton" src="img/rubyTag.png" />
+                  </a>
+                </li>
+              </ul>
+              <a href="#">
+                <img alt="seeProjectButton" src="img/Enabled.png" />
+              </a>
```

Comment:
- [ ] Hello, you've done well to include the `a` tag for the "See Project". however, it doesn't show up in your design please fix this issue. it might be better to simply use a `button` tag inside the `a` tag rather than using an image. 

|Your Design | Figma Design|
-|-
|![image](https://user-images.githubusercontent.com/18730106/153726573-54986b4d-93fc-4ba9-a8bb-17a81fa96fd3.png)|![image](https://user-images.githubusercontent.com/18730106/153726578-7e9c374a-2c09-4aab-aa99-d3747f20cac3.png)|